e2e-tests/playwright/e2e/learning-path-page.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { expect, test } from '@playwright/test';
+import { UIhelper } from '../utils/UIhelper';
+import { Common } from '../utils/Common';
+
+test.describe('Learning Paths', () => {
+  let common: Common;
+  let uiHelper: UIhelper;
+
+  test.beforeEach(async ({ page }) => {
+    uiHelper = new UIhelper(page);
+    common = new Common(page);
+    await common.loginAsGuest();
+  });
+
+  test('Verify that links in Learning Paths for Backstage opens in a new tab', async ({
+    page,
+  }) => {
+    test.slow();
```

Comment:
since you have added additional wait time to popup this no longer required. please remove.